assets/javascripts/kitten/components/tables/list-table/stories.js

```diff
@@ -141,6 +208,147 @@ const StyledCol8 = styled(ListTable.Col)`
   }
 `
 
+// Toggleable
+const linkclassNames = [
+  'k-u-color-primary1',
+  'k-u-decoration-none',
+  'k-u-size-tiny k-u-weight-regular',
+].join(' ')
+
+const defaultHeaderColWidth = {
+  default: '100%',
+  fromXs: '50%',
+  fromS: '25%',
+  fromL: '20%',
+}
+const defaultColWidth = {
+  default: '100%',
+  fromXs: '50%',
+  fromM: '25%',
+  fromL: '20%',
+}
+const defaultActionColWidth = { default: '100%', fromM: '50%', fromL: '20%' }
+
+const HeaderRow = styled(ListTable.Header)`
+  background-color: ${COLORS.background1};
+  color: ${COLORS.font1};
+  border-bottom: solid 2px ${COLORS.line1};
+  padding-left: ${pxToRem(CONTAINER_PADDING_THIN / 2)};
+  padding-right: ${pxToRem(CONTAINER_PADDING_THIN / 2)};
+
+  @media (min-width: ${ScreenConfig.M.min}px) {
+    padding-left: ${pxToRem(CONTAINER_PADDING / 2)};
+    padding-right: ${pxToRem(CONTAINER_PADDING / 2)};
+  }
+
+  ul {
+    padding: 0;
+  }
+`
+
+const Col = styled(({ align, width, ...props }) => (
+  <ListTable.Col {...props} />
+))`
+  min-height: ${pxToRem(90)};
+  flex-basis: auto;
+  flex-grow: 1;
+  width: ${({ width }) => (width ? width.default : '50%')};
+  font-size: ${stepToRem(-1)};
+
+  padding-left: ${pxToRem(CONTAINER_PADDING_THIN / 2)} !important;
+  padding-right: ${pxToRem(CONTAINER_PADDING_THIN / 2)} !important;
+
+  @media (min-width: ${ScreenConfig.XS.min}px) {
+    width: ${({ width }) => (width ? width.fromXs : '50%')};
+  }
+
+  @media (min-width: ${ScreenConfig.S.min}px) {
+    width: ${({ width }) => (width ? width.fromS : '20%')};
+
+    ${({ align }) =>
+      align === 'right' &&
+      css`
+        justify-content: flex-end;
+      `}
+  }
+
+  @media (min-width: ${ScreenConfig.M.min}px) {
+    width: ${({ width }) => (width ? width.fromM : '20%')};
+    padding-left: ${pxToRem(CONTAINER_PADDING / 2)} !important;
+    padding-right: ${pxToRem(CONTAINER_PADDING / 2)} !important;
+  }
+
+  @media (min-width: ${ScreenConfig.L.min}px) {
+    width: ${({ width }) => (width ? width.fromL : '20%')};
+  }
+
+  .nowrap {
+    white-space: nowrap;
+  }
+`
+
+const Row = styled(ListTable.Row)`
+  background-color: ${COLORS.background3};
+  padding-left: ${pxToRem(CONTAINER_PADDING_THIN / 2)};
+  padding-right: ${pxToRem(CONTAINER_PADDING_THIN / 2)};
+
+  @media (min-width: ${ScreenConfig.M.min}px) {
+    padding-left: ${pxToRem(CONTAINER_PADDING / 2)};
+    padding-right: ${pxToRem(CONTAINER_PADDING / 2)};
+  }
+
+  &.even {
```

Comment:
il faudrait voir si ça pourrait marcher : 

```suggestion
  &:nth-of-type(even) {
```